routes/runtime/github.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class CloverRuntime
+  hash_branch("github") do |r|
```

Comment:
question: sure, it makes sense to reject JWTs that are valid but refer to non-extant VMs (actually, the existence of these may suggest an attack, fair question if they should be logged or whether the logging opens a DOS vector). 

But do the JWTs also have a sensible expiry?

Finally, if the JWT is expired, vs. not-expired but carries a missing VM, do the error messages form any kind of meaningful information disclosure attack?

Reply:
If the request lacks a JWT, contains an invalid JWT, or references a non-existent VM/runner, we return the same response to prevent any potential information disclosure attacks.

https://github.com/ubicloud/ubicloud/blob/60e93097200eb1007445a84bd0b2e8622df7c2ed/clover_runtime.rb#L33
https://github.com/ubicloud/ubicloud/blob/60e93097200eb1007445a84bd0b2e8622df7c2ed/routes/runtime/github.rb#L5

Since runners are ephemeral, I didn't set an extra expiration for the JWT tokens. They expire automatically when the runner is deleted. However, the creation date is included in the payload. We can add a time limit to that whenever necessary.